src/utilities/ipfs.ts

```diff
@@ -0,0 +1,6 @@
+import fetch from 'cross-fetch';
+
+export const fetchIpfs = async (gateway: string, ipfsHash: string) => {
```

Comment:
Don't assume all gateways will follow this pattern. what if i needed to access http://localhost, or some gateways don't have `/ipfs` as the prefix.

Gateway should be the full ipfs endpoint